test/login.e2e-spec.ts

```diff
@@ -0,0 +1,75 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication, HttpStatus } from '@nestjs/common';
+import * as faker from 'faker';
+import * as request from 'supertest';
+import { factory, useRefreshDatabase, useSeeding } from 'typeorm-seeding';
+import { AppModule } from '../src/app.module';
+import { User } from '../src/entities/user.entity';
+
+describe('A user make a request to the authentication in the system', () => {
+  let app: INestApplication;
+
+  const makePayload = (payload?: { username?: string; password?: string }) =>
```

Comment:
Since I already requested a change to introduce `LoginDTO`, I suggest you using `Partial`:
```suggestion
  const makePayload = (payload?: Partial<LoginDTO>) =>
```

Reply:
Resolved on 113415a87c51f54d091cec5e7dc191829d53cf5d.